appveyor.yml

```diff
@@ -0,0 +1,36 @@
+clone_depth: 5
+environment:
+  nodejs_version: "7.8"
+platform: x64 # flow needs 64b platforms
+# Install scripts. (runs after repo cloning)
+install:
+  # When Yarn 0.28 will be on Chocolatey, we'll be able to do the following to
+  # install latest version:
+  # install yarn and node using Chocolatey
+  #  - choco install yarn
+  # But until then we need to install manually
+  # 1. Node
+  - ps: Install-Product node $env:nodejs_version $env:platform
+  # 2. Yarn is already at version 0.24.6, so let's keep it
+  # 3. Finally setup the project
+  - yarn install
+
+# Post-install test scripts.
+test_script:
+  # Output useful info for debugging.
+  - yarn versions
+  # run tests
+  - yarn license-check
+  - yarn lint
+  # Flow support for Windows isn't very stable, let's disable it for now
+  #- yarn flow -- --quiet
+  #- yarn flow -- stop
```

Comment:
I'll eventually add this line with "stop" in the script for circle ci too -- because the flow server stays behind after the check and takes some memory and CPU (for example `yarn build` below triggers a flow check)